src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const reversedRobot = {};
+  const valueCount = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    reversedRobot[value] = key;
+    valueCount[value] = (valueCount[value] || 0) + 1;
+  }
+
+  for (const count of Object.values(valueCount)) {
+    if (count > 1) {
+      return null;
+    }
+  }
```

Comment:
let's improve a little
1. no needs to invoke a loop twice
2. you can just use only one check
```suggestion
  for (const key in robot) {
    if (reversedRobot.hasOwnProperty(robot[key])) {
      return null;
    }

    reversedRobot[robot[key]] = key;
  }
```

Reply:
Oh, I got it 
Thank you!